api/openapi.yaml

```diff
@@ -30,228 +30,654 @@ tags:
+                    type: array
                 required:
                   - organizations
+                type: object
+          description: Ok. The response contains the list of organizations registered to Stalker.
         '401':
           $ref: '#/components/responses/Unauthorized'
         default:
           $ref: '#/components/responses/DefaultResponse'
-      summary: Read all organizations
-      security:
-        - Oauth scoped authorizations:
-            - user
-            - viewer
-    post:
+      summary: Read all organizations.
       tags:
+        - stalker mobile app
         - stalker web app
+    post:
       description: Request the creation of a new organization.
       operationId: postOrganizations
       requestBody:
-        required: true
         content:
           application/json:
             schema:
               $ref: ./model/OrganizationData.v1.yaml
+        required: true
       responses:
         '200':
-          description: The id of the newly created organization.
           content:
             application/json:
               schema:
                 $ref: ./model/GenericId.v1.yaml
+          description: Ok. The response contains the id of the newly created organization.
         '401':
           $ref: '#/components/responses/Unauthorized'
         '403':
           $ref: '#/components/responses/Forbidden'
         default:
           $ref: '#/components/responses/DefaultResponse'
-      summary: Create an organization
-  '/organizations/{organizationId}':
-    parameters:
-      - $ref: '#/components/parameters/organizationId'
-    put:
+      summary: Create an organization.
       tags:
         - stalker web app
-      operationId: putSpecificOrganization
-      description: "Update an organization's data."
+  /user/login:
+    post:
+      description: This endpoint should not have a security scheme.
+      operationId: postUserLoginCredentials
       requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: ./model/LoginData.v1.yaml
         required: true
+      responses:
+        '200':
+          content:
+            application/json:
+              schema:
+                $ref: ./model/User.v1.yaml
+          description: "Ok. The response contains the authenticated user's data."
+        '401':
+          content:
+            application/json:
+              schema:
+                properties:
+                  message:
+                    enum:
+                      - This email/password combination is not valid
+                    type: string
+                type: object
+          description: Unauthorized.
+        default:
+          $ref: '#/components/responses/DefaultResponse'
+      security: []
+      summary: Create a login request for a user.
+      tags:
+        - stalker mobile app
+        - stalker web app
+  /user/password/recovery:
+    post:
+      description: Request password recovery for an email.
```

Comment:
Io direi _Request the link for password recovery_ visto che è proprio questo che succede, chiedi che ti venga inviato il link alla procedura di password recovery